editor/filesystem_dock.cpp

```diff
@@ -3238,6 +3382,48 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, const Vect
 			p_popup->add_icon_item(get_editor_theme_icon(SNAME("NonFavorite")), TTR("Remove from Favorites"), FILE_REMOVE_FAVORITE);
 		}
 
+		if (p_paths.size() > 1 || p_paths[0] != "res://") {
+			HashSet<String> all_valid_conversion_to_targets = _get_valid_conversions_for_file_paths(p_paths);
+
+			int relative_id = 0;
+			if (all_valid_conversion_to_targets.size()) {
+				p_popup->add_separator();
+				// If we have more the one convert, collapse it into a submenu.
+				if (all_valid_conversion_to_targets.size() > 1) {
+					PopupMenu *new_menu = memnew(PopupMenu);
+					new_menu->set_name("Convert to");
+					new_menu->connect("id_pressed", callable_mp(this, &FileSystemDock::_generic_rmb_option_selected));
+
+					p_popup->add_child(new_menu);
+					p_popup->add_submenu_item(TTR("Convert to..."), "Convert to", FILE_NEW);
+
+					for (const String &E : all_valid_conversion_to_targets) {
+						Ref<Texture2D> icon;
+						if (has_theme_icon(E, SNAME("EditorIcons"))) {
+							icon = get_editor_theme_icon(E);
+						} else {
+							icon = get_theme_icon(E, SNAME("Resource"));
```

Comment:
Doesn't seem to be working for some reason 🤔
![image](https://github.com/user-attachments/assets/36046561-c840-4edb-bedc-221ed9e94e9f)

Reply:
What would the theme item of `E` in the `Resource` theme class mean?

Reply:
Ah yeah, it should be `get_editor_theme_icon(SNAME("Resource"));`

Reply:
Still wrong.

Reply:
I realized there is no icon for Resource, so it would be wrong either way.